docs/2.guide/4.recipes/3.custom-usefetch.md

```diff
@@ -90,28 +90,6 @@ const { data: modules } = await useAPI('/modules')
 </script>
 ```
 
-If you want to customize the type of any error returned, you can also do so:
-
-```ts
```

Comment:
We could also ensure that example codeblocks work by adding `twoslash` attribute.

Reply:
Emm.. I find that custom error type is hard for this now.. I want find some solution for that. And want more ease for this now.